ide/coqide/gen_gtk_platform.ml

```diff
@@ -0,0 +1,12 @@
+let to_string = function
+  | `X11 -> "X11"
+  | `QUARTZ -> "QUARTZ"
+  | `WIN32 -> "WIN32"
+
+let main () =
+  let out = open_out "gtk_platform.conf" in
```

Comment:
I feel like it's nicer to output to stdout, then only the dune file cares which filename is created

Reply:
Indeed, I guess I'm too tired now, thanks.

Reply:
I hope we can move that code completely out of Coq's configure by the way, #14189 almost does it.